rviz_ogre_vendor/CMakeLists.txt

```
@@ -116,7 +116,7 @@ macro(build_ogre)
   endif()
 
   if(WIN32)
-    set(OGRE_C_FLAGS "${OGRE_C_FLAGS} /w /EHsc")
+    set(OGRE_C_FLAGS "${OGRE_C_FLAGS} -std:c++14 /w /EHsc")
     set(OGRE_CXX_FLAGS "${OGRE_CXX_FLAGS} /w /EHsc")
```

Should it not be this instead?

```suggestion
    set(OGRE_C_FLAGS "${OGRE_C_FLAGS} /w /EHsc")
    set(OGRE_CXX_FLAGS "${OGRE_CXX_FLAGS} -std:c++14 /w /EHsc")
```

---

Certainly! I replicated my changes here for a pull request without build, didn't realize the change is different from my local copy of the file.